### ISSUE TITLE
Required parameter not propagated to fields classes

### ISSUE THREAD

**Issue by alichaddad** [js-next/js-ng#88]
```
class Typed(Field):
    def __init__(self, type_, default=None, indexed=False, required=False, **kwargs):
        self.type = type_
        self.default = default
        self.indexed = indexed

        # self.validate = Schema({

        # })
        super().__init__(default=default, indexed=indexed, **kwargs)

```

**Comment by xmonader** [comment]
@abom still relevant to the factory-improvement branch?

**Comment by abom** [comment]
Yes, fixed in #68.
